harness/determined/experimental/_native.py

```diff
@@ -392,3 +382,49 @@ def create_trial_instance(
     env, rendezvous_info, hvd_config = det._make_local_execution_env(False, config, hparams)
     trial_context = trial_def.trial_context_class(env, hvd_config)
     return trial_def(trial_context)
+
+
+def submit(
```

Comment:
Regarding naming, can we just remove the `det.experimental.create` and replace it with this functionality?  It's really unfortunate that we have that naming collision (although it seemed like a good idea when we first named it), but I'd much prefer that `det.experimental.create` matched the behavior of `det e create`.

Reply:
That sounds great. I'll make that change.